### PR TITLE
Matt/cross validation

### PR DESCRIPTION
**Pull request by mklienz**

### CODE REVIEW

File: hrpe/submit/create_submission.py [hunk 1]
```diff
@@ -55,9 +55,10 @@ def _validate_submissions_data(data, month, year):
 
     # Filter by time
     data = filter_data_by_time(data, time_start, time_end)
-    assert (
-        data.shape[0] == ndays * 48
-    ), f"The data must have the same number of rows as the number of days in the mont *48, instead it has {data.shape[0]}"
+    if month is not "september":
```
Comment (ari-nz) [inline hrpe/submit/create_submission.py:58]: Any reason for this?

File: pyproject.toml [hunk 2]
```diff
@@ -11,8 +11,9 @@ pandas = "^1.3.4"
 numpy = "^1.21.4"
 arrow = "^1.2.1"
 seaborn = "^0.11.2"
-u8darts = {extras = ["pdarima", "torch"], version = "^0.13.1"}
+u8darts = {extras = ["pmdarima", "torch"], version = "^0.13.1"}
```
Comment (ari-nz) [inline pyproject.toml:14]: huh? what a miss by us

**Review by ari-nz — APPROVED** [review]

File: hrpe/submit/create_submission.py (hunk 1 above)
Comment (mklienz) [reply hrpe/submit/create_submission.py:58]: Because the September test data only goes up to 10am 30th Sept lol

File: pyproject.toml (hunk 2 above)
Comment (mklienz) [reply pyproject.toml:14]: Right?

**Pull request merged**